cmd/cli/cmd/root_test.go

```diff
@@ -118,6 +118,7 @@ func NewTestBVNN(t *testing.T, defaultWorkDir string) (int, int) {
 	require.NoError(t, err)                           //
 	cfg.Accumulate.WebsiteEnabled = false             // Disable the website
 	cfg.Instrumentation.Prometheus = false            // Disable prometheus: https://github.com/tendermint/tendermint/issues/7076
+	cfg.Consensus.TimeoutCommit = time.Second / 10    // Increase block frequency
```

Comment:
I had thought the minimum was 1 second.

Reply:
I doubt 100ms is reasonable for any production system, but it doesn't break. I ran a quick test and the actual block frequency I get is more like 5Hz.